### PR TITLE
FISH-6772 javaee8 to jakartaee10

### PR DESCRIPTION
**Pull request by luiseufrasio**

### CODE REVIEW

File: bundles/fish.payara.eclipse.tools.server/plugin.xml [hunk 1]
```diff
@@ -922,6 +922,12 @@
             id="fish.payara.eclipse.tools.server.commands.OpenDomainHomeCommand"
             name="OpenDomainHomeCommand">
         </command>
+        <command
+			id="myplugin.newMenuItem"
+			name="New Menu Item"
+			description="This is a new menu item"
```
Comment (jGauravGupta) [inline bundles/fish.payara.eclipse.tools.server/plugin.xml:928]: Name and description are default that should be updated specific to feature.

**Review by jGauravGupta — CHANGES_REQUESTED** [review]

File: bundles/fish.payara.eclipse.tools.server/plugin.xml (hunk 1 above)
Comment (luiseufrasio) [reply bundles/fish.payara.eclipse.tools.server/plugin.xml]: Command removed!

File: bundles/fish.payara.eclipse.tools.server/src/fish/payara/eclipse/tools/server/handlers/MigrateHandler.java [hunk 2]
```diff
@@ -0,0 +1,152 @@
+package fish.payara.eclipse.tools.server.handlers;
```
Comment (jGauravGupta) [inline bundles/fish.payara.eclipse.tools.server/src/fish/payara/eclipse/tools/server/handlers/MigrateHandler.java:1]: Copyright missing

Comment (luiseufrasio) [reply bundles/fish.payara.eclipse.tools.server/src/fish/payara/eclipse/tools/server/handlers/MigrateHandler.java:10]: done!

**Pull request merged**